server-side/node-mssql-crud/src/controllers/warden.controller.js

```diff
@@ -22,4 +22,21 @@ export const gatepassApproveOrReject = async (req, res) => {
             res.status(500);
             res.send(error.message);
         }
+}
+
+
+export const gatepassCancelAndReject = async(req,res) => {
+    const approval_to = req.user.data.user_id
+    try{
+        const pool= await getConnection();
+        const result = await pool
+        .request()
+        .input("approval_to",sql.VarChar,approval_to)
+        .query(queries.getRejectedAndCancelledGatepass)
+
+    return res.json(result.recordsets[0])
```

Comment:
is this working (recordsets[0]) or should it be result.recodset ??

Reply:
It works, but for sakes of simplicity and overall code similarity, changed it to result.recordset

Reply:
result.recordset also works right??? please test once